packages/shared/src/lib/log.ts

```diff
@@ -150,11 +150,12 @@ export enum LogEvent {
   // 404 page
   View404Page = '404 page',
   // Source Actions - start
-  BlockSource = 'unfollow',
-  UnblockSource = 'follow',
-  // Source Actions - end
-  SubscribeSource = 'subscribe source',
-  UnsubscribeSource = 'unsubscribe source',
+  FollowSource = 'follow',
+  UnfollowSource = 'unfollow',
+  BlockSource = 'block',
+  UnblockSource = 'unblock',
+  SubscribeSource = 'subscribe',
+  UnsubscribeSource = 'unsubscribe',
```

Comment:
These 4 are not used anywhere in this PR from what I see in the diff?

Reply:
Did you agree somewhere else on this? https://github.com/dailydotdev/apps/pull/3385#discussion_r1689150558

Reply:
They where used since last pull request, now I've changed the values to reflect the new introduced events 👀